CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 1. New property operator: `filter` (#586, kudos to @iv-mexx)
 1. New operator `merge(with:)` (#600, kudos to @ra1028)
 1. New operator `map(value:)` (#601, kudos to @ra1028)
-1. `SignalProducer.merge` now accepts any combination of `SignalProducerConvertible` conforming types (#610, kudos to @1028)
+1. `SignalProducer.merge`, `SignalProducer.concat`, `SignalProducer.prefix`, `SignalProducer.then`, `SignalProducer.flatMapError` `Signal.flatMapError`, `Signal.withLatest` and `Property.init(initial:then:)` are now accept `SignalProducerConvertible` conforming types (#610, #611, kudos to @1028)
```

Comment:
The _are_ should be removed.

> `SignalProducer.merge`, `SignalProducer.concat`, `SignalProducer.prefix`, `SignalProducer.then`, `SignalProducer.flatMapError` `Signal.flatMapError`, `Signal.withLatest` and `Property.init(initial:then:)` now accept `SignalProducerConvertible` conforming types (#610, #611, kudos to @1028)

Reply:
@mdiep 
Sorry for the late response.
Done:+1: